import/webview/rawwebview.h

```diff
@@ -44,13 +47,16 @@ class RawWebView : public QuickMozView
     void virtualKeyboardMarginChanged();
     void contentOrientationChanged(Qt::ScreenOrientation orientation);
     void acceptTouchEventsChanged();
+    void openUrlInNewWindow(const QUrl &url);
```

Comment:
How are you planning to use this?

There are some cases where external links should be always opened to the browser application.

Reply:
The plan was to force target="_blank" on anchors, and then use to to invoke LinkHandler.handleLink(). It didn't work though because the target attribute didn't apply to implicit anchors.

So this doesn't solve my problem, but anchors that have target="_blank" are dead-ends in a web view at the moment so there might be some future value still.

Reply:
Agree with possible future value.

This is something that needs to be handled regardless of the current issue that you're looking.